rapthor/lib/cluster.py

```diff
@@ -3,118 +3,178 @@
 """
 import os
 import logging
-import sys
-import re
 import numpy as np
-import shutil
 
 log = logging.getLogger('rapthor:cluster')
 
 
-def get_total_memory():
+def get_available_memory():
     """
-    Returns the total memory in GB
+    Returns the available memory in GB
     """
-    tot_gb, used_gb, free_gb = list(map(int, os.popen('free -t -g').readlines()[-1].split()[1:]))
+    available_gb = list(map(int, os.popen('free -t -g').readlines()[1].split()[1:]))[0]
 
-    return tot_gb
+    return available_gb
```

Comment:
You may want to use `psutil.virtual_memory()` to get this info.

Reply:
Hmm, that does look nice, but would add another dependency. We can consider it if the current method seems to have issues.